src/main.tsx

```diff
@@ -3,10 +3,9 @@ import ReactDOM from "react-dom/client";
 import "./index.css";
 import { Provider } from "react-redux/es/exports";
 import store from "./modules/core/store";
-
-import { BrowserRouter, Route, Routes } from 'react-router-dom';
-
-import { ROUTER } from './modules/core/constants/navigation';
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import { ROUTER } from "./modules/core/constants/navigation";
+import "./modules/core/translation/i18n";
```

Comment:
Please don't include i18n configuration like this. Since it's not an import, you are just calling some logic in this case just by importing it.